src/dodal/devices/bimorph_mirrors/CAENels_bimorph_mirror_7_channel.py

```diff
@@ -0,0 +1,167 @@
+from ophyd import Component, EpicsSignal, EpicsSignalRO
+
+from dodal.devices.bimorph_mirrors.CAENels_bimorph_mirror_interface import (
+    CAENelsBimorphMirrorInterface,
+)
+
+
+class CAENelsBimorphMirror7Channel(CAENelsBimorphMirrorInterface):
+    """
+    Class representing a CAENels 7-Channel Bimorph Mirror.
+
+    Adds 7 channels to 0 inherited.
+
+    """
+
+    channel_1_voltage_target: EpicsSignal = Component(EpicsSignal, "C1:VTRGT")
+    channel_2_voltage_target: EpicsSignal = Component(EpicsSignal, "C2:VTRGT")
+    channel_3_voltage_target: EpicsSignal = Component(EpicsSignal, "C3:VTRGT")
+    channel_4_voltage_target: EpicsSignal = Component(EpicsSignal, "C4:VTRGT")
+    channel_5_voltage_target: EpicsSignal = Component(EpicsSignal, "C5:VTRGT")
+    channel_6_voltage_target: EpicsSignal = Component(EpicsSignal, "C6:VTRGT")
+    channel_7_voltage_target: EpicsSignal = Component(EpicsSignal, "C7:VTRGT")
+
+    channel_1_voltage_target_readback_value: EpicsSignalRO = Component(
+        EpicsSignalRO, "C1:VTRGT_RBV"
+    )
+    channel_2_voltage_target_readback_value: EpicsSignalRO = Component(
+        EpicsSignalRO, "C2:VTRGT_RBV"
+    )
+    channel_3_voltage_target_readback_value: EpicsSignalRO = Component(
+        EpicsSignalRO, "C3:VTRGT_RBV"
+    )
+    channel_4_voltage_target_readback_value: EpicsSignalRO = Component(
+        EpicsSignalRO, "C4:VTRGT_RBV"
+    )
+    channel_5_voltage_target_readback_value: EpicsSignalRO = Component(
+        EpicsSignalRO, "C5:VTRGT_RBV"
+    )
+    channel_6_voltage_target_readback_value: EpicsSignalRO = Component(
+        EpicsSignalRO, "C6:VTRGT_RBV"
+    )
+    channel_7_voltage_target_readback_value: EpicsSignalRO = Component(
+        EpicsSignalRO, "C7:VTRGT_RBV"
+    )
```

Comment:
Could: See [`EpicsSignalWithRBV`](https://nsls-ii.github.io/ophyd/generated/ophyd.areadetector.base.EpicsSignalWithRBV.html#ophyd.areadetector.base.EpicsSignalWithRBV) to make this bit a little nicer